parm/atm/variational/3dvar_dripcg.yaml

```diff
@@ -39,6 +39,6 @@ final:
     departures: anlmob
   increment:
     output:
-      filetype: latlon
-      filename: $(ANL_DIR)/atminc_latlon.
+      filetype: gaussian
```

Comment:
We shouldn't put this in develop of GDASApp until this is in develop of FV3-JEDI

Reply:
Make sense.  I replaced `gaussian` with `latlon`.    

Currently `latlon` output does not work ... but is in FV3-JEDI.   `gaussian` output works but isn't in FV3-JEDI.  fv3-jedi draft PR [#543](https://github.com/JCSDA-internal/fv3-jedi/pull/543) has been opened with a potential fix to `latlon` output.  fv3-jedi issue [#531](https://github.com/JCSDA-internal/fv3-jedi/issues/531) has been opened to document the addition of gaussian output.

Reply:
Updated at [2685761](https://github.com/NOAA-EMC/GDASApp/commit/268576170e506f938f68d65706751001ab40ee49)